config/settings.yml

```diff
@@ -974,6 +974,7 @@
     :keep_archived_quotas: 6.months
     :purge_window_size: 1000
 :product:
+  :name: null
```

Comment:
Just leave this blank

```yaml
:name: 
```

Reply:
Where will we put "Red Hat CloudForms" if this is the approach?  Right now we create a config/settings/production.yml that is layered on settings.yml, but I think it's layered *after* settings.yml.  If the user wants to "remove" that value, where do you think they will enter this value?

Reply:
The productization of this string is solved with `i18n.t` [here](https://github.com/ManageIQ/manageiq/pull/17773/files#diff-b37e88904167347189bb2e4a9a143876R24), so if this value is not set, it will behave as it behaves right now. Does this answer your question @Fryguy?